src/Funding/SFA.DAS.Funding.Api/Models/GetApprenticeshipsResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.Funding.Models;
+using System.Collections.Generic;
+
+namespace SFA.DAS.Funding.Api.Models
+{
+    public class GetApprenticeshipsResponse
+    {
+        public IEnumerable<ApprenticeshipDto> Apprenticeships { get; set; }
+
+        public GetApprenticeshipsResponse(List<Apprenticeship> source)
+        {
+            List<ApprenticeshipDto> apprenticeships = new();
+            foreach (var apprenticeship in source)
+            {
+                apprenticeships.Add(new ApprenticeshipDto()
+                {
+                    Uln = apprenticeship.Uln,
+                    FirstName = apprenticeship.FirstName,
+                    LastName = apprenticeship.LastName
+                });
+            }
+
+            Apprenticeships = apprenticeships;
```

Comment:
I maybe would have done this, up to you if you prefer it:

`Apprenticeships = source.Select(x => new ApprenticeshipDto
            {
                FirstName = x.FirstName,
                LastName = x.LastName,
                Uln = x.Uln
            });`

Reply:
Good point - fixed.